.github/workflows/docker.yaml

```diff
@@ -122,7 +122,7 @@ jobs:
       - docker
       - pick-connector-branch
     if: needs.docker.result == 'success'
-    uses: timescale/promscale/.github/workflows/go-e2e.yml@add_workflow_call
+    uses: timescale/promscale/.github/workflows/go-e2e.yml@${{ needs.docker.outputs.branch_name }}
```

Comment:
Should this be `needs.pick-connector-branch.outputs.branch`?

Reply:
thank you!